lib/lambda-cfn.js

```diff
@@ -24,6 +24,28 @@ lambdaCfn.lambdaSnsUser = lambdaSnsUser;
 lambdaCfn.lambdaSnsUserAccessKey = lambdaSnsUserAccessKey;
 lambdaCfn.outputs = outputs;
 lambdaCfn.load = load;
+lambdaCfn.getEnv = getEnv;
+
+function stripHyphens(r) {
+    return r.replace(/-/g, '');
+}
+
+var packages;
```

Comment:
@zmully any reason not to put these lines inside the `namespace` function itself?